<launch>
    <node pkg="prometheus_detection" type="aruco_det_v2" name="aruco_det_v2" output="screen">
        <param name="camera_topic" type="string" value="/prometheus/camera/rgb/image_raw" />
        <param name="camera_parameters" type="string" value="$(find prometheus_detection)/shell/calib_imx477_960x540.yaml" />
        <param name="output_topic" type="string" value="/prometheus/camera/rgb/image_aruco_det" />
        <param name="dictionary_type" type="int" value="6" />
        <param name="target_marker_length" type="double" value="0.0207" />
        <param name="calib_marker_length" type="double" value="0.0207" />
        <param name="calib_square_length" type="double" value="0.0345" />
    </node>
</launch>
